easybuild/framework/easyblock.py

```diff
@@ -1728,19 +1743,35 @@ def run_step(self, step, methods, skippable=False):
         """
         Run step, returns false when execution should be stopped
         """
-        if skippable and (self.skip or step in self.cfg['skipsteps']):
-            self.log.info("Skipping %s step" % step)
+        only_module = build_option('only_module')
+        force = build_option('force')
+        skip = False
```

Comment:
i would prefer you added a (private)? method `to_skip_or_not_skip`, have it return a boolean (and log message?) and use that. 

this rather crucial method is now cluttered with very odd logic, will also make unittesting easier i guess